docs/en/observability/apm/common-problems.asciidoc

```diff
@@ -54,11 +54,13 @@ If the `mapping.name.type` is `"text"`, your APM indices were not set up correct
 ----
 <1> The `service.name` `mapping.name.type` would be `"keyword"` if this field had been set up correctly.
 
+// Assuming the APM integration is no longer responsible for index mappings,
+// what is the fix now?
 To fix this problem, install the APM integration by following these steps:
 
---
-include::{observability-docs-root}/docs/en/observability/apm/getting-started-apm-server.asciidoc[tag=install-apm-integration]
---
+// --
+// include::{observability-docs-root}/docs/en/observability/apm/getting-started-apm-server.asciidoc[tag=install-apm-integration]
+// --
 
 This will reinstall the APM index templates and trigger a data stream index rollover.
```

Comment:
Same as above.

Reply:
same answer, I don't think this is a problem in 8.15.